pkg/rancher-desktop/components/SnapshotCard.vue

```diff
@@ -128,6 +128,8 @@ export default Vue.extend<Data, Methods, Computed, Props>({
     },
 
     async showRestoringSnapshotDialog() {
+      const snapshot = this.snapshot.name.length > 32 ? `${ this.snapshot.name.substring(0, 30) }...` : this.snapshot.name;
```

Comment:
The variable name should be `name` or `snapshotName`, but not `snapshot`. `this.snapshot` is an object, not a string, so the local var `snapshot` isn't strictly conflicting with it, but the types are different. But not in this PR, we need to merge it.